airflow/hooks/dbapi.py

```diff
@@ -345,11 +347,13 @@ def bulk_load(self, table, tmp_file):
         """
         raise NotImplementedError()
 
+    # TODO: Merge this implementation back to DbApiHook when dropping
+    # support for Airflow 2.2.
```

Comment:
This should be added to *OracleHook*’s `test_connection` instead.

Reply:
sorry, I misunderstood changes, now it should be better :)